src/containers/App.js

```diff
@@ -3,14 +3,13 @@ import { Router } from "react-router-dom";
 import history from "../routes/History";
 import Routes from "../routes/Routes";
 import "./App.scss";
-import Home from "./Home/Home";
 
 class App extends Component {
   render() {
     return (
       <>
         <Router history={history}>{Routes}</Router>
-        <Home />
+        {/* <Home /> */}
```

Comment:
Why did you comment the home components?

Reply:
This line should be deleted. Why? because if not, ```Home``` will be rendered with all routes not only in the home page.